docs/advanced_topics/third_party_tutorials.md

```diff
@@ -178,4 +178,4 @@ You can also find more resources from the community on [Awesome Wagtail](https:/
 ## Tip
 
 We are working on a collection of Wagtail tutorials and best practices.
-Please tweet [\@WagtailCMS](https://twitter.com/WagtailCMS) or [contact us directly](mailto:hello@wagtail.org) to share your Wagtail HOWTOs, development notes or site launches.
+Please share your Wagtail HOWTOs, development notes, or site launches in the [#watercooler](https://wagtailcms.slack.com/archives/C014L7KJH3N) channel on Slack, or feel free to reach out directly via [email](mailto:hello@wagtail.org).
```

Comment:
```suggestion
Please share your Wagtail HOWTOs, development notes, or site launches in the [Wagtail Slack workspace](https://github.com/wagtail/wagtail/wiki/Slack) in #watercooler, or feel free to reach out directly via [email](mailto:hello@wagtail.org).
```

That link is correct but will only work for people who are already on the Slack. I think it’ll be better for us to link to the page about joining the Slack.